ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -117,9 +143,35 @@ import UIKit
                                            paywallResultHandler: paywallResultHandler)
     }
 
+    @objc
+    public func presentPaywallIfNeeded(options: [String:Any],
+                                       paywallResultHandler: @escaping (String) -> Void) {
+        guard let requiredEntitlementIdentifier = options[PaywallOptionsKeys.requiredEntitlementIdentifier] as? String else {
+            print("Error: missing required entitlement identifier.")
+            return
+        }
+
+        let offeringIdentifier = options[PaywallOptionsKeys.offeringIdentifier] as? String,
+            displayCloseButton = options[PaywallOptionsKeys.displayCloseButton] as? Bool ?? false,
```

Comment:
Have to have defaults here so we don't modify `privatePresentPaywallIfNeeded` making  `displayCloseButton` nullable